rendezvous-server/src/handlers_to0.rs

```diff
@@ -203,6 +202,17 @@ pub(super) async fn ownersign(
         .await
         .map_err(Error::from_error::<messages::to0::OwnerSign, _>)?;
 
+    user_data
+        .store
+        .store_metadata(
+            &device_guid,
+            &fdo_store::MetadataKey::Ttl,
+            &chrono::Duration::from_std(Duration::from_secs(600))
```

Comment:
This Ttl is actually not just 600, but depends on the protocol value :).

Reply:
Damn, copy pasted , will fix it I see there’s a ttl variable indeed